skyve-war/src/main/java/modules/admin/Subscription/SubscriptionService.java

```diff
@@ -77,7 +83,8 @@ public static boolean anonymouslyUnsubscribe(Persistence persistence, String biz
 	 * @param bizCustomer
 	 * @return
```

Comment:
Clean up Javadoc; these parameters and return types should have proper descriptions.